mc_lib/tests/test_observable.py

```diff
@@ -77,3 +78,18 @@ def test_copying():
     assert_allclose(r1.mean, 1, atol=1e-15)
     assert_allclose(r2.mean, 2, atol=1e-15)
 
+def test_hdf5():
```

Comment:
decorate the test with `@pytest.mark.skipif(not HAVE_H5PY)` (see https://docs.pytest.org/en/latest/how-to/skipping.html for details).

Also, of course, will need to `import pytest`.